pkgs/top-level/all-packages.nix

```diff
@@ -3716,6 +3720,12 @@ in
 
   sstp = callPackage ../tools/networking/sstp {};
 
+  subsurface =
+    let callpkg = qt55.callPackage;
```

Comment:
`qt55.callPackage` is needed to resolve `qtbase`, `qttools`, etc. in `pkgs/applications/misc/subsurface/default.nix` - without that, `qt` packages are not defined properly. Please correct me if I am wrong.